website/docs/r/secure_cloud_auth_account.md

```diff
@@ -34,6 +34,9 @@ resource "sysdig_secure_cloud_auth_account" "sample" {
 
 * `component` - (Optional) The component configuration to enable on this cloud account. There can be multiple component blocks for a feature, one for each component to be enabled.
 
+* `regulatory_framework` - (Optional - AWS installs only) The type of Regulatory Framework for cloud account. Currently supported options are `REGULATORY_FRAMEWORK_UNSPECIFIED` and `RegulatoryFramework_REGULATORY_FRAMEWORK_US_FEDRAMP`.
+
```

Comment:
nit: can we keep the text consistent - and `REGULATORY_FRAMEWORK_US_FEDRAMP` ?

Reply:
done